docs_source_files/content/_index.en.md

```diff
@@ -33,21 +33,20 @@ import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
```

Comment:
Why the need to switch to Chrome?

Reply:
I'd leave everything with Firefox, since the other examples use Firefox as well.

Reply:
> I'd leave everything with Firefox, since the other examples use Firefox as well.

@diemol , i was checking code in chrome. Going further i will send code samples targeting FireFox